### PR TITLE
Add agenda points icon to relevant agenda displays

### PR DESCRIPTION
**Pull request by plural**
Screenshot 2021-05-30 7 22 27 PM
Screenshot 2021-05-30 7 22 41 PM
Screenshot 2021-05-30 7 23 17 PM

### CODE REVIEW

File: app/Resources/views/Search/display-zoom.html.twig [hunk 1]
```diff
@@ -29,7 +29,7 @@
                     </span>
                     <span class="card-cost">
                         {% if card.type_code == "agenda" %}
-                            {{ card.advancementcost }}/{{ card.agendapoints }}
+							{{ card.advancementcost }}/{{ card.agendapoints }}<span class="icon icon-agenda-points"></span>
```
Comment (NoahTheDuke) [inline app/Resources/views/Search/display-zoom.html.twig]: Looks like you used tabs instead of spaces

**Comment by NoahTheDuke** [comment]
This is cool! Looks a little odd to have a single icon (when both halves are associated with icons). Should you add in the "cost" icon to the other side?

**Comment by plural** [comment]
i don't have an advancement cost icon handy.  There isn't one in the visual assets set .  Using the credit icon isn't wrong, but it isn't right enough either.  :D

I'll ask on slack to see if i can get the asset and see how it looks.

File: app/Resources/views/Search/display-zoom.html.twig (hunk 1 above)
Comment (plural) [reply app/Resources/views/Search/display-zoom.html.twig]: done, thanks.

**Comment by plural** [comment]
i checked and we don't don't have an advancement icon to use.  I'm gonna go ahead and submit this.

**Pull request merged**